src/components/button/styles.module.css

```diff
@@ -0,0 +1,21 @@
+.container{
+    text-align: center;
+    padding-top: 1.5rem;
+}
+button{
```

Comment:
space before each class name, better to use a css formatter extension for consistency

Reply:
Done!